metadata.rb

```diff
@@ -16,8 +16,8 @@
 supports 'ubuntu', '>= 16.04'
 supports 'centos', '>= 7.3'
 
-recipe 'nexus-iq-server::default', 'Installs Nexus IQ Server and starts it as systemd service.'
-recipe 'nexus-iq-server::docker', 'Installs Nexus IQ Server. Instead of a systemd service start-nexus-iq-server.sh is provided in install_dir.'
```

Comment:
start-nexus-iq-server.sh is still ok, we use hyphens on the file system.

Reply:
Changed to use hyphens for file system names.